### ISSUE TITLE
Add comments for coded tendency terms to aid interpretation

### ISSUE THREAD

**Issue by BrodiePearson** [vanroekel/MPAS-Model#16]
The current tendency terms are decomposed into various terms that arise from a range of physical processes, e.g. https://github.com/vanroekel/MPAS-Model/blob/5dc8d6e40e31fb03492426e324d435637f6edf05/src/core_ocean/shared/mpas_ocn_adcReconstruct.F#L565-L569
Because these budgets involve several closure assumptions, the physical meaning of each of these tendency terms is not always obvious. We should comment the code to make it clear which physical processes are encompassed in different tendency terms. This will be based on @qingli411's [commented code.](https://github.com/vanroekel/MPAS-Model/blob/5dc8d6e40e31fb03492426e324d435637f6edf05/src/core_ocean/shared/mpas_ocn_adcReconstruct.F#L565-L569)

For future reference: changing the tendency terms, and adding them as save-able output variables requires the following steps:

1. Add the new tendency term in main ADC code: https://github.com/vanroekel/MPAS-Model/blob/103e68e300b7eb1fe27334e64fb2e402d9bd8b13/src/core_ocean/shared/mpas_ocn_adcReconstruct.F#L566
2. Add the new term in the ADC registry in 2 places: https://github.com/vanroekel/MPAS-Model/blob/103e68e300b7eb1fe27334e64fb2e402d9bd8b13/src/core_ocean/adc_mixing/Registry_adc_mixing_fields_opts.xml#L269-L270 https://github.com/vanroekel/MPAS-Model/blob/103e68e300b7eb1fe27334e64fb2e402d9bd8b13/src/core_ocean/adc_mixing/Registry_adc_mixing_fields_opts.xml#L576
3. Add the new term to `mpas_ocn_turbulence.F` in various places.
4. Add the new term to `mpas_ocn_vmix_adc.F` for deallocation: https://github.com/vanroekel/MPAS-Model/blob/103e68e300b7eb1fe27334e64fb2e402d9bd8b13/src/core_ocean/shared/mpas_ocn_vmix_adc.F#L531
5. If the new term is an output variable, add it to the `streams.ocean` file of the `forward` run directory

**Comment by vanroekel** [comment]
@BrodiePearson are you able to address this?  I think it would be good to have.  Note that the terms are somewhat explained in the adc registry file.

**Comment by BrodiePearson** [comment]
@vanroekel Yes, and I had not noticed their registry appearance so thanks for that! I'll try and do this in the next few days. I do have a question:

@katsmith133 & I discussed re-arranging/naming the tendency terms based on their physics (i.e. `w2tend_entrain`, `w2tend_transport`, `w2tend_buoyancy`, `w2tend_pressure_isotropy/w2tend_pressure_slow`, `w2tend_pressure_shear/w2tend_pressure_rapid`, `w2tend_splat`). **Would you support this approach?** This would reduce the amount of commenting required and also make it easier to directly compare budgets between the LES and ADC scheme (e.g. Issue #13)

**Comment by BrodiePearson** [comment]
I'm going to hold off on this issue until we figure out #18
